arch/arm/boot/dts/overlays/rpi-poe-overlay.dts

```diff
@@ -89,6 +79,52 @@
 		};
 	};
 
+	fragment@3 {
+		target = <&firmware>;
+		__overlay__ {
+			poe_pwm: pwm {
```

Comment:
Including `fw` somewhere in the label name (as used to be the case) would be clearer.

Reply:
Reverted the name. I'd thought I could change the thing pointed to via __overlay__ / __dormant__ so was looking for one generic name, but that doesn't work.